src/tests/system/tests/test_default_debug_level.py

```diff
@@ -0,0 +1,196 @@
+"""
+Automation for default debug level
+
+:requirement: SSSD - Default debug level
+"""
+
+from __future__ import annotations
+
+import re
+
+import pytest
+from pytest_mh.ssh import SSHProcessError
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopology, KnownTopologyGroup
+
+
+@pytest.mark.topology(KnownTopology.Client)
+def test_default_debug_level__check(client: Client):
+    """
+    :title: Check default debug level when sssd started successfully
+    :setup:
+        1. Clear logs and cache
+        2. Start SSSD with default debug level
+    :steps:
+        1. Check log files
+    :expectedresults:
+        1. "Starting with debug level = 0x0070" is in each file
+            Log message with number "0x3f7c0" is stored, if log contains more than one line
+
+    :customerscenario: False
+    """
+    client.sssd.common.local()
+    client.sssd.default_domain = "local"
+
+    client.sssd.clear(db=True, memcache=True, logs=True, config=False)
+    client.sssd.start(debug_level=None)
+
+    for log_filename in ["sssd", "sssd_local", "sssd_nss", "sssd_pam"]:
+        log_str = client.fs.read(f"/var/log/sssd/{log_filename}.log")
+        check = False
+
+        starting = re.compile(r"Starting with debug level = 0x0070")
+        assert starting.search(log_str) is not None, f"Log file has wrong format: {log_str}"
+
+        log_lines = log_str.split("\n")
+        if len(log_lines) <= 1:
+            continue
```

Comment:
Correct me if I'm wrong, but the test should fail if the number of lines is 0, and it shouldn't `continue` if the number of lines is 1 or higher than 1.

Reply:
No. The test checks content of these four files: "sssd", "sssd_local", "sssd_nss", "sssd_pam". On line 44  we check that each files contains the line: `Starting with debug level = 0x0070`. And the files that are longer that one line should contain line with "(0x3f7c0)".  "sssd" and "sssd_local" are longer that 1 line and the other two contain just one line.

Reply:
I see now that it's quite confusing. I will try to rewrite the testcase.

Reply:
I deleted whole nested for loop and now it's much more clear :smile: